lib/mixlib/shellout.rb

```diff
@@ -297,7 +289,6 @@ def parse_options(opts)
           self.password = setting
         when 'user'
           self.user = setting
-          self.with_logon = setting
```

Comment:
yea, and the `uid` and `gid` stuff can be breaking as well (even though I really don't think it does anything on windows)